# Aspreno

Python 3 global error handling & self-handling exception.
